acceptance_tests/features/steps/sample_loading.py

```diff
@@ -10,14 +10,14 @@
 from config import Config
 
 
-def get_emitted_cases_and_check_against_sample(sample_rows):
+def get_emitted_cases_and_check_against_sample(sample_rows, sensitive_columns=None):
```

Comment:
using `None` caused it to fail before. I don't think you've run all tests by running `make test`

Reply:
I suspect this should be `sensitive_columns=[]` otherwise it will bomb some of the scenarios